src/Game.ts

```diff
@@ -11,6 +12,14 @@ class Game {
     }
 
     afterMove(field: Field, move: string) {
+        for (let x = 0; x < this.board.boardSize; x++) {
+            for (let y = 0; y < this.board.boardSize; y++) {
+                if (this.board.fields[x][y].piece instanceof Pawn && (this.board.fields[x][y].piece as Pawn).isEnPassantPossible) {
+                    (this.board.fields[x][y].piece as Pawn).isEnPassantPossible = false;   
+                }
+            }
+        }
+
```

Comment:
1. Powinniśmy to wrzucić do nowej metody
2. Zobacz jak złapałem w jednej z metod poniżej wszystkie pionki danego koloru, może z tego skorzystać lub napisać coś na ten wzór?